optaweb-tsp-ui/pom.xml

```diff
@@ -0,0 +1,75 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>org.optaweb.tsp</groupId>
+	<artifactId>optaweb-tsp-ui</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<name>optaweb-tsp-ui</name>
+
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+		<version.frontend-maven-plugin>1.6</version.frontend-maven-plugin>
+		<java.version>1.8</java.version>
+		<version.maven-war-plugin>3.2.2</version.maven-war-plugin>
+		<node.version>v8.11.4</node.version>
+		<npm.version>6.4.1</npm.version>
+
+	</properties>
+
+	<build>
+		<resources>
+			<resource>
+				<!-- The webapp directory as generated by webpack -->
+				<directory>${project.basedir}/build</directory>
+				<!-- We put this into the static directory so that Spring Boot serves 
+					it through HTTP as static content -->
+				<targetPath>static</targetPath>
+			</resource>
+		</resources>
+		<finalName>optaweb-tsp-ui</finalName>
+		<plugins>
+			<plugin>
+				<groupId>com.github.eirslett</groupId>
+				<artifactId>frontend-maven-plugin</artifactId>
+				<version>${version.frontend-maven-plugin}</version>
```

Comment:
version goes in pluginManagement in root pom, see optaweb-employee-rostering root pom